### PR TITLE
add galley config validation metrics

### PR DESCRIPTION
**Pull request by ayj**

### CODE REVIEW

File: galley/pkg/crd/validation/webhook.go [hunk 1]
```diff
@@ -39,6 +40,58 @@ import (
 	"istio.io/istio/pkg/log"
 )
 
+var (
```
Comment (ozevren) [inline galley/pkg/crd/validation/webhook.go]: Minor nit: Could you put these in a "monitoring.go" file, to keep things organized? We've done this in Mixer:
https://github.com/istio/istio/blob/master/mixer/pkg/runtime/config/monitoring.go

**Comment by ayj** [comment]
Dup of https://github.com/istio/istio/pull/6827 (1.0 release branch)

**Pull request closed without merging**